nginx.conf

```
@@ -43,6 +43,11 @@ http {
             set $width $1;
             set $height $2;
         }
+ 
+        if ($uri ~ "^/([1-9][0-9]{0,2})xnone/") {
+            set $width $1;
+            set $height -;
```

Should this be
```suggestion
            set $height '-';
```

I'm not 100% clear on nginx config syntax.

---

i've tested this and it seems to work with "-" and '-'